src/libpriv/rpmostree-passwd-util.c

```diff
@@ -409,6 +410,8 @@ rpmostree_check_passwd_groups (gboolean         passwd,
             goto out;
           
           old_path = g_file_resolve_relative_path (root, commit_filepath);
+          if (!g_file_load_contents (old_path, cancellable, &old_contents, NULL, NULL, error))
+            goto out;
```

Comment:
Any reason we can't use `glnx_file_get_contents_utf8_at` here? Or just keep the previous logic of only setting `old_path` and changing the `gs_file_load_contents_utf8` in the deleted hunk below to `glnx_file_get_contents_utf8_at` ?